### DISCUSSION TITLE
Documentation for Getting Started

### DISCUSSION THREAD

**Discussion by CarlSargunar** [CommunityToolkit/Maui#166] [Q&A] [answered]
Hi,

It would be really nice to see some documentation on getting started with this repo, and how to contribute.

- Setup required to build
- Notes on how to test maybe?
- Some issues tagged with "good first issue"

Happy to spend some time putting that documentation together myself is there is anyone able and willing to walk me through getting the library set up and compiled and ready for testing

**Comment by TheCodeTraveler** [comment, marked as answer]
Agreed! We are working on adding the docs to [docs.microsoft.com/dotnet/communitytoolkit](https://docs.microsoft.com/dotnet/communitytoolkit) (they're not available just yet, though). 

In the meantime, here's some blog posts to help you get started: 
- [Using .NET MAUI Community Toolkit](https://codetraveler.io/2021/10/18/using-net-maui-community-toolkit-2/)
- [Contributing to .NET MAUI Community Toolkit](https://devblogs.microsoft.com/dotnet/contributing-to-net-maui-community-toolkit/?WT.mc_id=mobile-0000-bramin)

**Reply by CarlSargunar** [reply]
Awesome - thanks @brminnick - Look forward to seeing those

**Comment by CliffAgius** [comment]
Closed as answered
